.github/workflows/slack-cli-generate-docs.yaml

```diff
@@ -0,0 +1,43 @@
+name: Generate Slack CLI Docs
+
+on:
+  workflow_dispatch:
+
+jobs:
+  update-docs:
+    runs-on: ubuntu-latest
+
+    steps:
+
+    - name: Install Slack CLI
+      run: |
+        curl -fsSL https://downloads.slack-edge.com/slack-cli/install.sh | bash
+
+    - name: Generate a GitHub token
+      id: ghtoken
+      uses: actions/create-github-app-token@v2
```

Comment:
💔 todo: We should use pinned versions here though I don't think the healthscore has released such a check yet...

📫 note: I can soon add these versions if needed!